### PR TITLE
Ghost barter fix. Ferro cost down. Skune fix. Landlocked Cooldown

### PR DESCRIPTION
**Pull request by farafrog44**
Reduced the prices of ferroslimes:
5 poudrins to 2
2 cans to 1
No more string 

Hopefully fixed ghost bartering and skune not being on other people's slime checks.

### CODE REVIEW

**Comment by magicksRampage** [comment]
Relates to issues: #74 and #72

If you fix skune poudrins too, we can close #74 too.

**Comment by magicksRampage** [comment]
also contains a fix for #76

**Comment by farafrog44** [comment]
I apologise for fucking up a lot lol

File: ewmap.py [hunk 1]
```diff
@@ -1924,6 +1924,17 @@ async def loop(cmd):
 		response = "You need to be on the edge of the map to !loop through it. Try a district bordering an outskirt, the ferry, or Slime's End Cliffs."
 		return await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 	else:
+		mutation_data = EwMutation(id_mutation=ewcfg.mutation_id_landlocked, id_user=cmd.message.author.id, id_server=cmd.message.guild.id)
+		
+		if len(mutation_data.data) > 0:
+			time_lastuse = int(mutation_data.data)
```
Comment (magicksRampage) [inline ewmap.py:1930]: you don't actually set this data anywhere. you have to set it with the current time, after a !loop is actually initialized

Comment (magicksRampage) [reply ewmap.py:1930]: this is fine now

**Pull request closed without merging**